### PR TITLE
Add pact testing

### PR DESCRIPTION
**Pull request by cobbal**
refs: MBL-13999
affects: none
release note: none

Test plan:

Run tests for scheme "pact", and verify tmp/pacts/canvas-ios-canvas_lms_api.json has some stuff in it.

Manually uploading pacts to the pact broker is future work and will not be super-automated.

### CODE REVIEW

File: Core/PactTests/Tests/PactTests.swift [hunk 1]
```diff
@@ -0,0 +1,65 @@
+//
+// This file is part of Canvas.
+
```
Comment (cobbal) [inline Core/PactTests/Tests/PactTests.swift]:
```suggestion
```

File: Podfile [hunk 2]
```diff
@@ -53,6 +53,11 @@ abstract_target 'defaults' do
   pod 'AFNetworking', '~> 3.0'
   pod 'Mantle', '~> 1.5.5'
 
+  target 'CoreTests' do
```
Comment (cobbal) [inline Podfile]:
```suggestion
  target 'PactTests' do
```

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Please add a description for Danger.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Please run `yarn update-headers`. The following do not have the correct license header: 
* Core/PactTests/Tests/PactTests.swift
  </td>
    </tr>
  </tbody>
</table>



Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.57% | 93.57% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/0uy7udc) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F0uy7udc)
teacher: [Link](https://rebrand.ly/xz0tgxx) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fxz0tgxx)
parent: [Link](https://rebrand.ly/9m0warl) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F9m0warl)

**Pull request merged**